### ISSUE TITLE
Add tensorboard to neural implementations

### ISSUE THREAD

**Issue by nsorros** [wellcometrust/WellcomeML#103]
[Tensorboard](https://www.tensorflow.org/tensorboard/get_started) is a really useful tool when training a neural network to monitor the loss and experiments. It can be simply added as a callback in the fit method.

To display the GUI someone would need to run separately tensorboard.

**Comment by nsorros** [comment]
It has already been added to BiLSTM and CNN but not in other like BertSemanticEquivalence. Let's break to individual tickets or create a checklist.
